tests/tests_unit/test_cognite_client.py

```diff
@@ -93,20 +93,18 @@ def test_project_is_correct(self, rsps_with_login_mock):
         assert c.config.project == "test"
 
     def test_no_api_key_no_token_set(self):
-        with unset_env_var(["COGNITE_API_KEY", "COGNITE_TOKEN_URL"]):
+        with unset_env_var("COGNITE_TOKEN_URL"):
             with pytest.raises(
                 CogniteAPIKeyError, match="No API key or token or token generation arguments have been specified"
```

Comment:
should we edit this error message to not include "No API key" anymore?

Reply:
No, legacy auth is still enabled for many tenants. and will be for some time. so sdk still supports authenticating with api keys.